### DISCUSSION TITLE
Can't enable ssl

### DISCUSSION THREAD

**Discussion by Coolpakz** [ant-media/Ant-Media-Server#6716] [Q&A]
There is a problem with the script. Please re-run the enable_ssl.sh script.

Tried on 3 vps's.
After installing and try enabling the ssl, I always got this problem.

**Comment by muratugureminoglu** [comment]
Hi @Coolpakz 

Could you please take a look at the discussion link below? If it not work for you, please continue to the discussion thread.

https://github.com/orgs/ant-media/discussions/6720

Thanks.

**Comment by Coolpakz** [comment]
Nah! I'm not using ant-media anymore.
